src/components/super-tabs.ts

```diff
@@ -689,7 +689,13 @@ export class SuperTabs implements OnInit, AfterContentInit, AfterViewInit, OnDes
 
   // needed to make Ionic Framework think this is a tabs component... needed for Deeplinking
   setTabbarPosition() {}
-
+ 
+ // update the segment button widths manually 
+  indexSegmentButtonWidths() {
+    setTimeout(() => {
```

Comment:
ensures that rendering time is provided when data changes,it's okay to reduce,in my case,i set 1 second delay